src/wp-includes/blocks.php

```diff
@@ -760,6 +760,108 @@ function get_hooked_blocks( $name ) {
 	return $hooked_blocks;
 }
 
+function insert_hooked_blocks( $block ) {
+	$hooked_blocks = get_hooked_blocks( $block['blockName'] );
+	foreach ( $hooked_blocks as $hooked_block_type => $relative_position ) {
+		$hooked_block = array(
+			'blockName'    => $hooked_block_type,
+			'attrs'        => array(),
+			'innerHTML'    => '',
+			'innerContent' => array(),
+			'innerBlocks'  => array(),
+		);
+		// Need to pass full current block, (possibly its parent), relative position, and hooked_block.
+		$block = insert_hooked_block( $hooked_block, $relative_position, $block );
+	}
+	return $block;
+}
```

Comment:
This will need https://github.com/WordPress/wordpress-develop/pull/5242 to work with `serialize_blocks`.

```suggestion
function insert_hooked_blocks( $block, $parent ) {
	$hooked_blocks = get_hooked_blocks( $block['blockName'] );
	foreach ( $hooked_blocks as $hooked_block_type => $relative_position ) {
		$hooked_block = array(
			'blockName'    => $hooked_block_type,
			'attrs'        => array(),
			'innerHTML'    => '',
			'innerContent' => array(),
			'innerBlocks'  => array(),
		);

		$block = insert_hooked_block( $parent, $hooked_block, $relative_position, $block );
	}
	return $block;
}
```